extras/rya.streams/query-manager/src/test/java/org/apache/rya/streams/querymanager/QueryManagerIT.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.rya.streams.querymanager;
+
+import org.junit.Before;
+
+public class QueryManagerIT {
```

Comment:
You shouldn't need to write integration test to test the query manager. You can mock everything and just write unit tests.

Reply:
Yeah, I named it before i thought about what was going to need to be tested and how